### ISSUE TITLE
SW Moonwell - Tailoring : Felcloth to Mooncloth not working

### ISSUE THREAD

**Issue by Jinmcx** [chromiecraft/chromiecraft#1304]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [ ] Horde

### Content Phase:

- [X] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

### Current Behaviour

Can't purify fel cloth into moon cloth.

### Expected Blizzlike Behaviour

Able to purify fel cloth into moon cloth.

### Steps to reproduce the problem

1. Go to Stormwind Mooncloth Location
2. Attempt to purify felcloth into mooncloth - [gives error requirement moonwell]

Worked at Felwood/Ashenvale moonwell.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/3d4befd7043fbbceb32f7c2a6e0a4f9f638f8b0a

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-Zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by Jinmcx** [comment]
![WoWScrnShot_072721_172205](https://user-images.githubusercontent.com/19652090/127241657-7e06082e-95e1-4c80-a675-3dcd68f04dce.jpg)

**Comment by Synful-Syn** [comment]
A bit of history about this bug...

A few weeks ago, I noticed some meat (called Moonwell) were in that moonwell. I asked on Discord (channel #testing ) if they were supposed to be visible, I had no real response.
![image](https://user-images.githubusercontent.com/22718174/127250596-324447bf-0769-4871-bc63-b0f59dc6a8c9.png)
https://discord.com/channels/788063059926712341/790877776692510741/865748017796481034

A few days later, I saw in the ChromieCraft patch notes (#changelog-chromie) that they were removed.
> Several haunches of meat have been removed from the Stormwind City Moonwell.

https://discord.com/channels/788063059926712341/794247581763895316/868441225609490442

It says "removed" instead of "turned invisible".
